examples/gopherlouse 4.mapping

```diff
@@ -0,0 +1,10 @@
+p26:h10
```

Comment:
Duplicate -- it seems like these duplicates might have been created my merge? Please be a little careful when you merge.

Reply:
Removed.